### PR TITLE
Improvement/farmv1

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/components/ClassicElasticTab.tsx [hunk 1]
```diff
@@ -59,20 +138,65 @@ function ClassicElasticTab() {
...
         |
       </Text>
 
+      {showLegacyExplicit && (
+        <>
+          <MouseoverTooltip text={notSupportedMsg || ''}>
+            <Flex
+              sx={{ position: 'relative' }}
+              alignItems={'center'}
+              onClick={() => {
```
Comment (nguyenhoaidanh) [inline src/components/ClassicElasticTab.tsx:157]: use handleSwitchTab

File: src/components/ClassicElasticTab.tsx [hunk 2]
```diff
@@ -59,20 +138,65 @@ function ClassicElasticTab() {
...
+          <Text fontWeight={500} fontSize={[18, 20, 24]} color={theme.subText} marginX={'12px'}>
+            |
+          </Text>
+        </>
+      )}
+
       <Flex
         alignItems={'center'}
         onClick={() => {
           const newQs = { ...qs, tab: VERSION.CLASSIC }
```
Comment (nguyenhoaidanh) [inline src/components/ClassicElasticTab.tsx:195]: use handleSwitchTab

File: src/components/YieldPools/ElasticFarmGroup/Row.tsx [hunk 3]
```diff
@@ -156,6 +156,16 @@ const Row = ({
...
 
+  const numberOutRangePos = depositedPositions.filter(
+    pos => pos.pool.tickCurrent < pos.tickLower || pos.pool.tickCurrent >= pos.tickUpper,
+  ).length
+
+  const numberInRangePos = depositedPositions.filter(
+    pos => pos.pool.tickCurrent >= pos.tickLower && pos.pool.tickCurrent < pos.tickUpper,
+  ).length
+
+  console.log(numberInRangePos, numberOutRangePos)
```
Comment (nguyenhoaidanh) [inline src/components/YieldPools/ElasticFarmGroup/Row.tsx:167]: rm

Comment (viet-nv) [reply src/components/YieldPools/ElasticFarmGroup/Row.tsx:167]: Done

File: src/components/ClassicElasticTab.tsx (hunk 2 above)
Comment (viet-nv) [reply src/components/ClassicElasticTab.tsx]: Done

File: src/components/ClassicElasticTab.tsx (hunk 1 above)
Comment (viet-nv) [reply src/components/ClassicElasticTab.tsx:157]: Done

File: src/pages/ProAmmPool/index.tsx [hunk 4]
```diff
@@ -118,7 +117,7 @@ export default function ProAmmPool() {
 
   const debouncedSearchText = useDebounce(searchValueInQs.trim().toLowerCase(), 300)
 
-  const [showClosed, setShowClosed] = useState(true)
```
Comment (nguyenhoaidanh) [inline src/pages/ProAmmPool/index.tsx:121]: intent or not

Comment (viet-nv) [reply src/pages/ProAmmPool/index.tsx:121]: intent

Comment (viet-nv) [reply src/pages/ProAmmPool/index.tsx:121]: user complains about it. then product changes requirement. LOL

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**